packages/sui-test/bin/cypress/cypress.json

```diff
@@ -0,0 +1,10 @@
+{
+    "fixturesFolder": false,
+    "pluginsFile": false,
+    "supportFile": false,
+    "videoRecording": false,
+    "screenshotOnHeadlessFailure": false,
+    "trashAssetsBeforeHeadlessRuns": true,
+    "viewportWidth": 1240,
```

Comment:
what about mobile?

Reply:
This is just the default. You can change the context in your specs with `cy.viewport`
https://docs.cypress.io/api/commands/viewport.html#Syntax